frontend/src/lib/components/TaxonomicFilter/TaxonomicFilter.tsx

```diff
@@ -49,7 +53,14 @@ export function TaxonomicFilter({
 
     return (
         <BindLogic logic={taxonomicFilterLogic} props={taxonomicFilterLogicProps}>
-            <div className={`taxonomic-filter${taxonomicGroupTypes.length === 1 ? ' one-taxonomic-tab' : ''}`}>
+            <div
+                className={clsx(
+                    'taxonomic-filter',
+                    taxonomicGroupTypes.length === 1 && 'one-taxonomic-tab',
+                    !width && 'force-minimum-width'
+                )}
+                style={{ height: `${height}px`, width: `${width}px`, display: 'flex', flexDirection: 'column' }}
```

Comment:
I don't think we should be creating CSS like `style="height:px;width:px;display:flex..."` that this will produce.